pkgs/data/fonts/ricty/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchurl, google-fonts, migu, fontforge, which }:
+
+stdenv.mkDerivation rec {
+  name = "ricty-${version}";
+  version = "4.1.0";
+
+  src = fetchurl {
+      url = "http://www.rs.tus.ac.jp/yyusa/ricty/ricty_generator-${version}.sh";
+      sha256 = "1cv0xh81fi6zdjb62zqjw46kbc89jvwbyllw1x1xbnpz2il6aavf";
+  };
+
+  unpackPhase = ''
+    install -m 0770 $src ricty_generator.sh
+  '';
+
+  patchPhase = ''
+    sed -i 's/fonts_directories=".*"/fonts_directories="$inconsolata $migu"/' ricty_generator.sh
+  '';
+
+  buildInputs = [ google-fonts migu fontforge which ];
+
+  buildPhase = ''
+    inconsolata=${google-fonts} migu=${migu} ./ricty_generator.sh auto
```

Comment:
Since this build takes external `buildInputs`, that might be updated. We cannot use `outputHash`. Otherwise the package would break, whenever somebody updates `google-fonts`.

Reply:
After all this is only a performance optimization for large font packages.

Reply:
Thank you. I misunderstood that.